### PR TITLE
A&S [Fall 2020 and After] data

### PR DESCRIPTION
**Pull request by einc**
### Summary 

Added A&S [Fall 2020 and after] college data. It's very similar to A&S [before Fall 2020] except for the distribution requirements. 

- [] slacked on #cp-dev, but would we want to still have general warning functionality (even if requirements aren't self check). I couldn't add a checker that checks that "students can only double-count distribution requirements on a maximum of two courses" for distribution requirements

### Test Plan 

- Test the A&S [Fall 2020 and after] college data. AP/Exam credit should work without any breaking changes. 

Screen Shot 2021-03-22 at 9 40 34 PM

### CODE REVIEW

File: src/requirements/data/colleges/asPreFA2020.ts [hunk 1]
```diff
@@ -119,6 +119,7 @@ const casPreFA2020Requirements: readonly CollegeOrMajorRequirement[] = [
       },
     },
   },
+  // TODO: AP/Test Credit cannot be applied to Distribution Requirements
```
Comment (tcho6319) [inline src/requirements/data/colleges/asPreFA2020.ts:123]: you might be able to use Ben's flag for `disallowTransferCredit`

File: src/requirements/data/colleges/asFA2020.ts [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+      //   fulfilledBy: 'self-check',
+      // },
+    },
+  },
+  // TODO: do not check students can only double-count distribution requirements on a maximum of two courses
+  {
+    name: 'Distribution Requirements',
+    description:
+      'A minimum of 8 courses must be taken to fulfill all distribution categories:' +
+      ' ALC-AS, BIO-AS, ETM-AS, GLC-AS, HST-AS, PHS-AS, SCD-AS, SSC-AS, SMR-AS. ' +
```
Comment (tcho6319) [inline src/requirements/data/colleges/asFA2020.ts:134]: Seem to be missing SDS-AS in description. Could be confusing for user

File: src/requirements/data/colleges/asFA2020.ts [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+          'SINHA 2***',
+          'SPAN 2***',
+          'SWAHL 2***',
+          'TAG 2***',
+          'THAI 2***',
+          'TURK 2***',
+          'VIET 2***',
+          'YORUB 2***',
+          'ZULU 2***'
+        ),
```
Comment (tcho6319) [inline src/requirements/data/colleges/asFA2020.ts:75]: I don't think this check includes courses above 2000-level.

**Review by tcho6319 — CHANGES_REQUESTED** [review]
Left some nits. 

1. A bigger issue I found was that the A&S [Fall 2020 and After] doesn't seem to have any majors connected to it. Idk if this is a fix you can do in the requirements data.

https://user-images.githubusercontent.com/54298311/112166670-35f58380-8bc6-11eb-9c46-2af3a49f2761.mov

File: src/requirements/data/colleges/asPreFA2020.ts (hunk 1 above)
Comment (einc) [reply src/requirements/data/colleges/asPreFA2020.ts:123]: > Left some nits.
> 
> 1. A bigger issue I found was that the A&S [Fall 2020 and After] doesn't seem to have any majors connected to it. Idk if this is a fix you can do in the requirements data.
> 
>  Screen.Recording.2021-03-23.at.10.54.03.AM.mov

Oh yup i forgot to add them, fixing that. TY!

Comment (einc) [reply src/requirements/data/colleges/asPreFA2020.ts:123]: and yes I added his flag in the other PR <3 removing comment now!

**Review by tcho6319 — APPROVED** [review]
LGTM

**Pull request merged**